modules/monitoring-managing-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="managing-core-platform-alerting-rules_{context}"]
+= Managing alerting rules for core platform monitoring
+
+:FeatureName: User-defined and user-modified alerting rules for core platform monitoring
```

Comment:
Not sure if we can drop the `user-defined` here without creating more confusion. Happy to leave this as is, but maybe someone has a good idea.

Reply:
I changed it to `Adding and modifying alerting rules for core platform monitoring`